packages/core/docs/changelog/5442_2.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  description: 'Updated the default logger to use matching console log levels, like debug, info, warn, error.',
+  link: 'https://github.com/vuestorefront/vue-storefront/pull/5442',
+  isBreaking: false,
```

Comment:
```suggestion
  isBreaking: true,
```
it is a breaking change for the core